signer/tests/integration/postgres.rs

```diff
@@ -3176,6 +3116,7 @@ async fn should_process_withdrawals() {
     testing::storage::drop_db(store).await;
 }
 
+#[ignore = "some test fixtures need some work before this can be enabled"]
```

Comment:
mmm mixed feeling about ignoring the pg tests as well, do we need to?

Reply:
Ahhh! This is me being sloppy. It's not supposed to be here

Reply:
https://github.com/stacks-network/sbtc/pull/1559/commits/a408682acea83df37b859bc68179a01cf19a666a